system-addon/content-src/asrouter/asrouter-content.jsx

```diff
@@ -129,12 +170,15 @@ export class ASRouterUISurface extends React.PureComponent {
         shouldSendImpressionOnUpdate={shouldSendImpressionOnUpdate}
         // This helps with testing
         document={this.props.document}>
-          <SimpleSnippet
-            {...this.state.message}
-            UISurface="NEWTAB_FOOTER_BAR"
-            getNextMessage={ASRouterUtils.getNextMessage}
-            onBlock={this.onBlockById(this.state.message.id)}
-            sendUserActionTelemetry={this.sendUserActionTelemetry} />
+          <LocalizationProvider messages={generateMessages(this.state.message.content.text)}>
+            <SimpleSnippet
+              {...this.state.message}
+              richText={<RichText text={this.state.message.content} />}
```

Comment:
I would do `text={this.state.message.content.text} links={this.state.message.content.links}` or just `{...this.state.message.content}`, passing message.content as text is kind of confusing